### PR TITLE
Release v1.5.0

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Create `ui-plugin-find-organization` release
https://issues.folio.org/browse/UIPFO-5
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "babel-eslint": "^9.0.0",
     "eslint": "^5.6.0",
     "eslint-plugin-filenames": "^1.3.2",
+    "eslint-plugin-jest": "^23.0.4",
     "eslint-plugin-no-only-tests": "^2.3.0",
```
Comment (aliaksei-chumakou) [inline package.json]: "eslint-plugin-react-hooks": "^2.1.1",

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -52,31 +47,27 @@
...
 - Does this introduce breaking changes?
-  - [ ] Were any API paths or methods changed, added or removed?
-  - [ ] Were there any schema changes?
-  - [ ] Did any of the interface versions change?
-  - [ ] Were permissions changed, added, or removed?
-  - [ ] Are there new interface dependencies?
-  - [ ] There are no breaking changes in this PR.
-  
+  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
+  - [*] There are no breaking changes in this PR.
```
Comment (aliaksei-chumakou) [inline PULL_REQUEST_TEMPLATE.md]: please remove the asterisk

File: package.json [hunk 3]
```diff
@@ -25,14 +25,17 @@
...
+    "eslint-plugin-react-hooks": "^2.1.1",
     "react": "^16.5.1",
     "react-dom": "^16.5.1",
     "react-intl": "^2.7.2",
     "react-redux": "^5.0.7",
     "redux": "^4.0.0"
   },
   "dependencies": {
+    "@folio/stripes-acq-components": "^1.2.0",
```
Comment (aliaksei-chumakou) [inline package.json]: I don't think we should add this

Comment (AndreiShumski2) [reply package.json]: removed it

**Pull request merged**